package.json

```diff
@@ -40,6 +40,7 @@
     "cookie-parser": "^1.4.6",
     "express": "^4.17.1",
     "express-session": "^1.17.3",
+    "ffbinaries": "^1.1.5",
```

Comment:
I don't think we should include this dependency because it comes with something like 100 more dependencies. Most of them are coming from [request](https://www.npmjs.com/package/request) which is deprecated anyway.

I haven't gone through it yet but looking at the library we should be able to set this up ourselves using our current libraries https://github.com/ffbinaries/ffbinaries-node/blob/master/ffbinaries-lib.js

Reply:
Yeah, my intention was to only include ffbinaries as an intermediate step. 

I was intending to do a rewrite of ffbinaries after this. I also noticed the dependency on request . The library is also written using callbacks rather than promises, and is full of synchronous function calls (although it's probably OK since it's a startup code). I think that I saw some fork that replaces request with axios, but I haven't looked closely yet, since it was convenient to use ffbinaries as-is as a starting point.

Are you ok with including it as an interim step? I think it would take another week or two to rewrite it for our purposes with no/very few additional dependencies. That's definitely next on my todo list.

Reply:
I made a quick run at it but ran out of time today to polish it up more. It should be working though.

I did not implement the progress ticker for the download but shouldn't be difficult if we want to implement that.

I removed `locateBinariesSync` function since it wasn't being used.

`downloadBinaries`, `getVersionData` and `listVersions` are all promises now.

Reply:
Very nice (and quick!).